src/Microsoft.FeatureManagement/ReadSessionResult.cs

```diff
@@ -0,0 +1,18 @@
+﻿namespace Microsoft.FeatureManagement
+{
+    /// <summary>
+    /// The result of reading the feature state for a session.
+    /// </summary>
+    public struct ReadSessionResult
```

Comment:
Why don't we use nullable bool instead?

Reply:
Something threw me off with nullable bool. Perhaps because there once was a discussion to have feature manager return nullable bool as a meaning that the state of a feature couldn't be determined. 

But, nullable bool could work here. It's simpler. We can go with that.